src/cclib/io/wfxwriter.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""A writer for wfx format files."""
+
+import os.path
+
+from . import filewriter
+
+class WFXWriter(filewriter.Writer):
+    """A writer for wfx files."""
+
+    required_attrs = ['atomcoords', 'atomnos', 'gbasis', 'charge', 'homos']
+    keyword = 'GTO'
```

Comment:
In the past, ADF only used Slater-type orbitals. I assume that's still the case, and if so, doesn't this mean the WFX writer won't work with ADF?